README.md

```diff
@@ -353,6 +353,34 @@ as follows:
 }
 ```
 
+If you are not using NixOS you can place the following flake in ``~/.config/nixpkgs/flake.nix``
+to load your standard home-manager config:
+
+```nix
+{
+  description = "A Home-Manager flake";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    home-manager.url = "github:nix-community/home-manager";
+  };
+
+  outputs = inputs: {
+    homeConfigurations = {
+      jdoe = inputs.home-manager.lib.homeManagerConfiguration rec {
+        system = "x86_64-linux";
+        homeDirectory = "/home/${username}";
+        username = "jdoe";
+        configuration = { config, lib, pkgs, ... }:
+          {
+            imports = [ ./home.nix ];
+          };
```

Comment:
```suggestion
        configuration = ./home.nix;
```

While it's good that it shows that the `configuration` parameter is a module, I think it can be simplified to the above.

Reply:
I think the original one is better as such small things where the things that I had problems with when I started using Nix.

Reply:
I'm inclined to agree with @berbiche, seems unlikely that one would want to put configuration inside the flake file. I can only imagine that it would be useful if you want to import multiple files but then something like

    configuration.imports = [ ./file1.inx ./file2.nix ];

should suffice.

Reply:
That is what I was actually trying to do myself but did not achieve.